src/OrchardCore.Modules/OrchardCore.Settings/SiteSettings.cs

```diff
@@ -1,11 +1,22 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
 using Microsoft.AspNetCore.Routing;
+using Newtonsoft.Json.Linq;
 using OrchardCore.Documents;
+using OrchardCore.Entities;
 
 namespace OrchardCore.Settings
 {
     // When updating class also update SiteSettingsDeploymentSource and SettingsStep.
-    public class SiteSettings : DocumentEntity, ISite
+    public class SiteSettings : DocumentEntity, ICacheableEntity, ISite
     {
+        private static readonly JObject _defaultProperties = new();
```

Comment:
I think this is a bad idea. if you have multiple instances of `SiteSettings`, and you set the `Properties = null`, now you have multiple instances of the `SiteSettings` that share the same static JObject which is bad.

Reply:
There's a difference between `_properties` & `_defaultproperties`, the first one SHOULD NOT be `static`, right?

Reply:
yes. `_defaultproperties` should not be static. You should just keep it `?? new Object()`. It's simple enough.

Reply:
IMHO using `static` field is better than creating new object for every instance